pkg/bootstrap/bootstrap_config_test.go

```diff
@@ -146,11 +149,17 @@ func TestGolden(t *testing.T) {
 		out = "/tmp"
 	}
 
+	credPath := out + "/sd_cred.json"
+	if err := ioutil.WriteFile(credPath, []byte(`{"type": "service_account", "project_id": "my-sd-project"}`), os.ModePerm); err != nil {
+		t.Fatalf("unable write file: %v", err)
+	}
+	os.Setenv("GOOGLE_APPLICATION_CREDENTIALS", credPath)
```

Comment:
This is one hell of a bug waiting to happen in prow. If you don't unset it, it might really screw all kinds of libraries in GCP.

Reply:
good catch.